server/routes/prePrisonEducation/update/index.ts

```diff
@@ -24,6 +24,15 @@ export default (router: Router, services: Services) => {
   const qualificationDetailsUpdateController = new QualificationDetailsUpdateController()
   const qualificationsListUpdateController = new QualificationsListUpdateController(educationAndWorkPlanService)
 
+  router.get('/prisoners/:prisonNumber/education/add-qualifications', [
```

Comment:
This is the new route that sets up the `EducationDto` in the prisoner context, then redirects to `qualification-level` to allow the CIAG start start entering the 1st qualification